### DISCUSSION TITLE
PWAsForFirefox 1.4.2

### DISCUSSION THREAD

**Discussion by filips123** [filips123/PWAsForFirefox#159] [Announcements]
## Fixed

* Fixes launch bug from the previous release (#157).
* Fixes PWAs not using CSD when they should.

<hr /><em>This discussion was created from the release <a href='https://github.com/filips123/PWAsForFirefox/releases/tag/v1.4.2'>PWAsForFirefox 1.4.2</a>.</em>

**Comment by quiffboy** [comment]
Which version should I download for a new M1 Macbook?

**Reply by filips123** [reply]
You need to update it using Homebrew:

```sh
brew update
brew upgrade firefoxpwa
```

**Reply by quiffboy** [reply]
Worked perfectly, many thanks.
